holdings_maintenance/pds4/pds4infoshelf.py

```
@@ -0,0 +1,931 @@
+        except KeyError:
+            logger.error('Missing entry in checksum file', abspath)
+            checksum = ''
+
+        size = (0,0)
+        ext = os.path.splitext(abspath)[1]
+        if ext.lower() in PREVIEW_EXTS:
+            try:
+                im = Image.open(abspath)
+                size = im.size
+                im.close()
+            except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)